appveyor.yml

```diff
@@ -8,7 +8,7 @@ clone_folder: c:\Cosmos
 
 init:
 - cmd: |
-    git clone https://github.com/CosmosOS/IL2CPU.git c:\IL2CPU --depth 1
+    git clone https://github.com/CosmosOS/IL2CPU.git c:\IL2CPU --depth 1 --branch crossplatform 
```

Comment:
we should configure to run tests on both linux and windows but for now just run the windows tests

Reply:
@valentinbreiz make sure to revert this change